extensions/wikia/AttributeServiceMWApi/AttributeServiceMWApiController.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+class AttributeServiceMWApiController extends WikiaController {
+
+    /**
```

Comment:
Bah! After spilling water on my old laptop I'm still setting up this new one...guess I missed that part. I'll update.